xcode/Subconscious/Shared/Components/Common/Profile/FollowUserSheet.swift

```diff
@@ -99,8 +99,8 @@ struct FollowUserSheetModel: ModelProtocol {
                         .petnameField(.setValue(input: petname.verbatim))
                     )
                 }
-                .catch { error in
-                    Just(FollowUserSheetAction.failToFindUniquePetname)
+                .recover { _ in
+                    FollowUserSheetAction.failToFindUniquePetname
```

Comment:
Nit: I often like to include the error's message in the fail action, so I can log it in the fail handler.